components/badge/script.js

```diff
@@ -0,0 +1,11 @@
+const inputOne=document.querySelector("#btn-ide");
+
+function myFunction(){
+    const userInput=document.querySelector("#toggle1");
```

Comment:
You can change the variable name and name it something relatable, as there is no user input here, it leads to confusion